saskatoon/member/admin.py

```diff
@@ -96,9 +96,9 @@ class AuthUserAdmin(UserAdmin):
                     'is_core',
                     'is_admin',
                     'is_active',
+                    # 'has_temporary_password',
```

Comment:
please remove comment, I think the `UserIsOnboarding` filter is enough, no need for `has_temporary_password` in list_display

Reply:
removed